.github/workflows/trymerge.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           check-latest: false
           cache: pip
           architecture: x64
-      - run: pip install pyyaml==6.0 rockset==0.8.10
+      - run: pip install pyyaml==6.0 rockset==0.8.10 six==1.16.0
```

Comment:
Please don't. Six should be long dead in 2023

Reply:
updated rockset version to get rid of six dependency